templates/web/base/common_header_tags.html

```diff
@@ -6,6 +6,10 @@
     id="js-meta-data" data-page="[% page %]"
     data-cobrand="[% c.cobrand.moniker %]">
 
+[% IF csrf_token %]
+  <meta content="[% csrf_token %]" name="csrf-token" />
```

Comment:
Yes, those changes (from a commit 22nd March) postdate my comment of 17th March :)